mongo/integration/unified/unified_spec_runner.go

```diff
@@ -26,6 +26,39 @@ var (
 		// GODRIVER-1773: This test runs a "find" with limit=4 and batchSize=3. It expects batchSize values of three for
 		// the "find" and one for the "getMore", but we send three for both.
 		"A successful find event with a getmore and the server kills the cursor (<= 4.4)": {},
+		// TODO(CSOT): Unskip these CSOT spec tests once createChangeStream operations are correctly formatted.
+		"timeoutMS can be configured on a MongoClient - createChangeStream on client":         {},
+		"timeoutMS can be set to 0 on a MongoClient - createChangeStream on client":           {},
+		"timeoutMS can be configured on a MongoClient - createChangeStream on database":       {},
+		"timeoutMS can be set to 0 on a MongoClient - createChangeStream on database":         {},
+		"timeoutMS can be configured on a MongoClient - createChangeStream on collection":     {},
+		"timeoutMS can be set to 0 on a MongoClient - createChangeStream on collection":       {},
+		"timeoutMS can be configured on a MongoDatabase - createChangeStream on database":     {},
+		"timeoutMS can be set to 0 on a MongoDatabase - createChangeStream on database":       {},
+		"timeoutMS can be configured on a MongoDatabase - createChangeStream on collection":   {},
+		"timeoutMS can be set to 0 on a MongoDatabase - createChangeStream on collection":     {},
+		"timeoutMS can be configured on a MongoCollection - createChangeStream on collection": {},
+		"timeoutMS can be set to 0 on a MongoCollection - createChangeStream on collection":   {},
+		// TODO(CSOT): Unskip these CSOT spec tests once estimatedDocumentCount correctly expects an aggregate instead of a count command.
```

Comment:
See my [comment](https://github.com/mongodb/specifications/pull/960#discussion_r861074075) in the spec PR:
> Tests for `estimatedDocumentCount` expect a `count` `commandStartedEvent` in this PR. Depending on the wire version/whether drivers have implemented DRIVERS-2228, an `aggregate` may be started for `estimatedDocumentCount`. I think tests with `estimatedDocumentCount` will need some `min/maxServerVersion` logic.